src/embind/emval.js

```diff
@@ -201,8 +201,13 @@ var LibraryEmVal = {
       argsList += (i!==0?", ":"")+"arg"+i; // 'arg0, arg1, ..., argn'
     }
 
+    function getMemory() {
```

Comment:
Please add a comment, like `We must pass in the memory views to the new Function() because the body cannot see the outer scope where the views are defined. However, memory growth may refresh the views, so we cannot pass in their current values, and instead pass in a function to get the latest view`.